test/integration/__utils__/withGitIntegration.js

```diff
@@ -81,6 +81,9 @@ export const withGitIntegration =
     // Init repository with initial commit
     await utils.execGit('init')
```

Comment:
We could specify the initial branch name here instead:

```suggestion
    await utils.execGit(['init', '--initial-branch, 'master'])
```

Reply:
Sure, that's a similar solution. Addressed in e9d86de.